### PR TITLE
Service accounts examples

### PR DESCRIPTION
**Pull request by awkoren**
For: https://cloud.google.com/iam/docs/creating-managing-service-accounts

### CODE REVIEW

**Comment by awkoren** [comment]
TODO: Don't construct the Service Account email by requiring project-id and name. Just make them give the full email.

File: iam/api/ServiceAccounts/Options.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+// limitations under the License.
+
+using CommandLine;
+
+namespace GoogleCloudSamples
+{
+    [Verb("create", HelpText = "Creates a service account.")]
+    internal class CreateServiceAccountOptions
+    {
+        [Option("project-id", HelpText = "The project Id.", Required = true)]
```
Comment (seanosterberg) [inline iam/api/ServiceAccounts/Options.cs]: Update "The project Id" to "The project ID"

File: iam/api/ServiceAccounts/Options.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        public string Name { get; set; }
+
+        [Option("display-name", HelpText = "The service account's friendly display name.")]
+        public string DisplayName { get; set; }
+    }
+
+    [Verb("list", HelpText = "Lists service accounts.")]
+    internal class ListServiceAccountOptions
+    {
+        [Option("project-id", HelpText = "The project Id.", Required = true)]
```
Comment (seanosterberg) [inline iam/api/ServiceAccounts/Options.cs]: Update "The project Id" to "The project ID"

File: iam/api/ServiceAccounts/Options.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        public string ProjectId { get; set; }
+    }
+
+    [Verb("rename", HelpText = "Updates a service account's display name.")]
+    internal class RenameServiceAccountOptions
+    {
+        [Option("project-id", HelpText = "The project Id.", Required = true)]
```
Comment (seanosterberg) [inline iam/api/ServiceAccounts/Options.cs]: Update "The project Id" to "The project ID"

File: iam/api/ServiceAccounts/Options.cs [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        public string Name { get; set; }
+
+        [Option("name", HelpText = "The service account's new friendly display name.", Required = true)]
+        public string DisplayName { get; set; }
+    }
+
+    [Verb("delete", HelpText = "Deletes a service account.")]
+    internal class DeleteServiceAccountOptions
+    {
+        [Option("project-id", HelpText = "The project Id.", Required = true)]
```
Comment (seanosterberg) [inline iam/api/ServiceAccounts/Options.cs]: Update "The project Id" to "The project ID"

File: iam/api/ServiceAccounts/Options.cs [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    [Verb("rename", HelpText = "Updates a service account's display name.")]
+    internal class RenameServiceAccountOptions
+    {
+        [Option("project-id", HelpText = "The project Id.", Required = true)]
+        public string ProjectId { get; set; }
+
+        [Option("name", HelpText = "The service account name.", Required = true)]
+        public string Name { get; set; }
+
+        [Option("name", HelpText = "The service account's new friendly display name.", Required = true)]
```
Comment (seanosterberg) [inline iam/api/ServiceAccounts/Options.cs]: Duplicate "name" flags -- change this one to "display-name"

File: iam/api/ServiceAccounts/Program.cs [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+            }
+            // [END iam_list_service_accounts]
+            return 0;
+        }
+
+        public static int RenameServiceAccount(string projectId, string name,
+            string newDisplayName)
+        {
+            // [START iam_rename_service_account]
+            // First, get a ServiceAccount using List() Get()
```
Comment (seanosterberg) [inline iam/api/ServiceAccounts/Program.cs]: Do you mean just Get()?

**Review by seanosterberg — CHANGES_REQUESTED** [review]
LGTM with a few changes. Also, remember to remove the --project-id flag from rename() and delete(), opting for the `-` wildcard instead.

**Pull request merged**